src/modules/deal-guarantee/deal-guarantee.service.create-guarantees.test.ts

```diff
@@ -113,19 +105,20 @@ describe('DealGuaranteeService', () => {
       expect(guaranteeCreatedInAcbs.GuaranteeType.GuaranteeTypeCode).toBe(PROPERTIES.DEAL_GUARANTEE.DEFAULT.guaranteeTypeCode);
     });
 
-    it(`replaces effectiveDate with today's date if effectiveDate is before today`, async () => {
-      when(currentDateProviderGetLatestDateFromTodayAnd).calledWith(effectiveDateAsDate).mockReturnValueOnce(today);
+    it(`replaces effectiveDate with today's date if effectiveDate is after today`, async () => {
```

Comment:
Thanks for fixing this @avaitonis - can we update the description of the `effectiveDate` field in the DTO for the deal guarantee POST as well please, it currently describes the old logic